ghcup-vanilla-0.0.8.yaml

```diff
@@ -4906,6 +4906,122 @@ ghcupDownloads:
+              dlSubdir: ghc-9.10.2-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.2/ghc-9.10.2-x86_64-deb12-linux.tar.xz
+            unknown_versioning: *ghc9102-x86_64-linux-deb11
+          Linux_Fedora:
+            '>= 33': &ghc9102-x86_64-linux-fedora33
+              dlHash: a35bdc8a594b4b5fa59a07167f80279cba545f2378b763998bd91a9e2a210d0e
+              dlSubdir: ghc-9.10.2-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.2/ghc-9.10.2-x86_64-fedora33-linux.tar.xz
+          Linux_Mint:
+            < 20: &ghc9102-x86_64-linux-ubuntu18_04
+              dlHash: 77c7536cd7624c76b110cf6bd84fb2a1b954ea23f0dca6a35c625159f2f33967
+              dlSubdir: ghc-9.10.2-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.2/ghc-9.10.2-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc9102-x86_64-linux-ubuntu20_04
+              dlHash: 1ecbf53954dd6a3e5fd8bbb50134d5e0412bbd7c25157cfb82e4f1b889e232b1
+              dlSubdir: ghc-9.10.2-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.2/ghc-9.10.2-x86_64-ubuntu20_04-linux.tar.xz
+            unknown_versioning: *ghc9102-x86_64-linux-ubuntu20_04
+          Linux_RedHat:
+            < 9: &ghc9102-x86_64-linux-rocky8
+              dlHash: 51d52d287052226341c46bf57e4dd7b196715695138259db386821490afdbab4
+              dlSubdir: ghc-9.10.2-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.2/ghc-9.10.2-x86_64-rocky8-linux.tar.xz
+            '>= 9': *ghc9102-x86_64-linux-fedora33
+            unknown_versioning: *ghc9102-x86_64-linux-fedora33
+          Linux_Ubuntu:
+            ( >= 16 && < 18 ): *ghc9102-x86_64-linux-deb9
+            ( >= 18 && < 19 ): *ghc9102-x86_64-linux-ubuntu18_04
+            unknown_versioning: *ghc9102-x86_64-linux-ubuntu20_04
+          Linux_UnknownLinux:
+            unknown_versioning: *ghc9102-x86_64-linux-rocky8
+          Windows:
+            unknown_versioning:
+              dlHash: 7677132b72e6b8d99ed7e3d30f19845efaea348a7016afd3558a48cfe208b60e
+              dlSubdir: ghc-9.10.2-x86_64-unknown-mingw32
+              dlUri: https://downloads.haskell.org/~ghc/9.10.2/ghc-9.10.2-x86_64-unknown-mingw32.tar.xz
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlHash: a236cfc42ea7280a691dc595313bd6b87efc4bd1c43dfe96b22194108fd81c1c
+              dlSubdir: ghc-9.10.2-aarch64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.10.2/ghc-9.10.2-aarch64-apple-darwin.tar.xz
+          Linux_Alpine:
+            unknown_versioning:
+              dlHash: e54530aecb6bfbe8971a1344ba9e9193540efe7722b59146418007eeaf290c9c
+              dlSubdir: ghc-9.10.2-aarch64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.2/ghc-9.10.2-x86_64-alpine3_18-linux.tar.xz
```

Comment:
This is incorrect (x86_64).  Is there a bug in the metadata generation script?

Reply:
Yes. I've fixed it manually in the metadata for now and will put up/backport a patch to fix the generation script.